Calibration/PCCAlCaRecoProducers/test/PCC_Random_cfg.py

```diff
@@ -0,0 +1,210 @@
+#########################
+#Author: Sam Higginbotham
+#Purpose: To investigate the AlCaPCCProducer input and output. 
+#########################
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("ALCARECO")
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/eos/cms/store/data/Run2015D/AlCaLumiPixels/ALCARECO/LumiPixels-PromptReco-v4/000/260/039/00000/1CF2A210-5B7E-E511-8F4F-02163E014145.root','/eos/cms/store/data/Run2015D/AlCaLumiPixels/ALCARECO/LumiPixels-PromptReco-v4/000/260/039/00000/1E2B0829-707E-E511-B51B-02163E0145FE.root','/eos/cms/store/data/Run2015D/AlCaLumiPixels/ALCARECO/LumiPixels-PromptReco-v4/000/260/039/00000/2666E76A-707E-E511-92E4-02163E014689.root','/eos/cms/store/data/Run2015D/AlCaLumiPixels/ALCARECO/LumiPixels-PromptReco-v4/000/260/039/00000/2A1E3304-707E-E511-946C-02163E014241.root')
+)
+
+#Added process to select the appropriate events 
+process.OutALCARECOPromptCalibProdPCC = cms.PSet(
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOPromptCalibProdPCC')
+    ),
+    outputCommands = cms.untracked.vstring('drop *', 
+        'keep *_alcaPCCProducer_*_*', 
+        'keep *_MEtoEDMConvertSiStrip_*_*')
+)
+#Make sure that variables match in producer.cc and .h
+process.alcaPCCProducer = cms.EDProducer("AlcaPCCProducer",
```

Comment:
Yes I think it is expected to run in a central workflow. Do I need to make files similar to 
https://github.com/cms-sw/cmssw/blob/master/Calibration/TkAlCaRecoProducers/python/AlcaBeamSpotProducer_cfi.py 
Along with _cff.py? If so, how can I specify the High Level trigger so I can select between Random and ZeroBias HLT filters?